packit/config.py

```diff
@@ -175,6 +177,11 @@ def get_context_settings() -> dict:
     )
 
 
+class RunCommandType(Enum):
+    openshift = "openshift"
+    cli = "cli"
```

Comment:
how about renaming cli to "local", because it really is about running locally, in current environment; cli is the end-product using the functionality, which sounds confusing to me

Reply:
`local` is fine. And for running POD in Openshift, I would use `sandcastle` instead of `openshift`